src/assessor_tools.yml

```diff
@@ -65,10 +103,31 @@
         python2: yes
         unarchive_extra_opts:
           - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/BC-SECURITY/Empire/tarball/master"
+        csharp: yes
+        install_dir: /tools/Empire
+        pip_packages:
+          - .
+        powershell: yes
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/FortyNorthSecurity/EXCELntDonut/\
```

Comment:
There are probably other janky dependencies in these tools as well.  My general rule has been to install dependencies where the tool repo does things in a standard way and let the assessors otherwise handle the installation.  In this case, though, I created commit 0d8dfc8 to handle the additional dependency.

Reply:
Unfortunately the change does not fully duplicate the logic in the `install.sh` script, but I'm not sure if you want to implement the `mv EXCELntDonut/CLRvoyance/CLRvoyance/* EXCELntDonut/CLRvoyance` at the end (the full command is
```
git clone https://github.com/Accenture/CLRvoyance EXCELntDonut/CLRvoyance && mv EXCELntDonut/CLRvoyance/CLRvoyance/* EXCELntDonut/CLRvoyance
```
). I'm not sure how you want to proceed with this, but since you mentioned your general rule I'm ok with reverting this addition if you don't want to fool with fully replicating the `install.sh` script.

Reply:
I un-`tar` the [Accenture/CLRvoyance](https://github.com/Accenture/CLRvoyance) tarball into `/tools/EXCELntDonut/EXCELntDonut/CLRvoyance` with `--strip-components=1`, which I think reproduces the effect of the `mv` command.  What am I missing?

Reply:
The `install.sh` script would be moving the contents of `/tools/EXCELntDonut/CLRvoyance/CLRvoyance` up to `/tools/EXCELntDonut/CLRvoyance`. It's a dirty, dirty hack to add `CLRvoyance` as a module within the `EXCELntDonut` package.

Reply:
Please have a look at commit 952b62d, which I believe does the correct thing.

Reply:
Makes sense. Sorry for the extra work!

Reply:
No problem.  It was necessary work.